src/NuGet.Services.EndToEnd/Support/PackageProperties.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.Services.EndToEnd.Support
+{
+    public class PackageProperties
+    {
+        public bool IsSymbolsPackage { get; }
+
+        public HashSet<string> IndexedFiles { get; }
+
+        public PackageProperties() { }
+
+        public PackageProperties(bool isSymbolsPackage, HashSet<string> indexedFiles)
+        {
+            IsSymbolsPackage = isSymbolsPackage;
+            IndexedFiles = indexedFiles;
+        }
+
+        public static PackageProperties Default()
```

Comment:
Why have this if the default ctor is public?

Reply:
removed.